modules/nlu/src/backend/language/language-provider.ts

```diff
@@ -131,7 +140,7 @@ export class RemoteLanguageProvider implements LanguageProvider {
           this._validProvidersCount++
           data.languages.forEach(x => this.addProvider(x.lang, source, client))
 
-          this.extractLangServerVersion(data)
+          this.extractLangServerInfo(data)
```

Comment:
just realized that this is in a for loop (a user can define multiple language providers)